libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -63,10 +71,11 @@ type subsystem interface {
 }
 
 type Manager struct {
-	mu       sync.Mutex
-	Cgroups  *configs.Cgroup
-	Rootless bool // ignore permission-related errors
-	Paths    map[string]string
+	mu         sync.Mutex
+	Cgroups    *configs.Cgroup
+	Rootless   bool // ignore permission-related errors
+	Paths      map[string]string
+	Subsystems subsystemSet
```

Comment:
Is this set somewhere? I cannot find it.

Reply:
I've added it as a way to customize the controller and allow to specify what subsystems are supported/used.
If it is `!= nil`, then `getSubsystems()` will use it: https://github.com/opencontainers/runc/pull/2113/files#diff-3638867461edf7257846ceba1724fb8aR142

Thinking more of it, probably it is not that useful to have so I can drop it if you prefer

Reply:
I went forward and dropped it.